docs/API/defineFetch.md

```diff
@@ -0,0 +1,83 @@
+| `make`        | func (...keyArgs) | required\* | The make function creates a key to reference data and returns a function used for `useFetch` and `useDispatch` hooks. <br /><br /> The `keyArgs` arguments are optional strings that can be used when creating the data store key. A `keyArg` would be needed for fetching and storing data with unique ids. |
+| `share`       | object            | -          | `share` object grants other `defineFetches` the ability to share the same key on the store. This will allow `defineFetch`'s to override onSuccess requests. This is ideal for CRUD endpoints that need to `get` and `edit` the same piece of data.                                                           |
+| `conflict`    | string            | 'cancel'   |                                                                                                                                                                                                                                                                                                              |
+
+&nbsp;
+
+### `defineFetch.make(...) Return Props`
+
+---
+
+| Prop      | Type                                                                        | Default    | Description                                                                                                                                                                                                                                                                                                                                   |
+| --------- | --------------------------------------------------------------------------- | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `key`     | string[]                                                                    | required\* | These strings will be used to create a key for the data store.                                                                                                                                                                                                                                                                                |
+| `request` | func (...requestArgs)<br /> Returns -> <br /> func (...resiftServices) <br> | required\* | The `request` function is what will dispatch. The return data is what will be stored and returned in `useFetch`. <br /><br /> `requestArgs` are optional arguments needed for a request. <br /><br /> `resiftServices` are services such as `http` and `dispatch` supplied through Resift that give additional functionality to your request. |
```

Comment:
@ricokahler, what are the standards on arguments like these?

Reply:
Any arguments that are required to make a request but aren't required to identify the fetch should go in the `request` args, otherwise (if they identify the request) then they should go in the `make` args